tests/h/search/core_test.py

```diff
@@ -163,6 +164,7 @@ def UriCombinedWildcardFilter(self, patch):
         return patch("h.search.core.query.UriCombinedWildcardFilter")
 
 
+@pytest.mark.usefixtures("group_service", "moderation_service")
```

Comment:
This isn't necessary as these are already included by the index fixture which is included by the Annotation fixture.

Reply:
Ah, I see. I've updated the tests so that the search tests only directly use the `group_service` fixture and the indexing fixtures only directly use the `moderation_service` fixture.